src/track/beatfactory.cpp

```diff
@@ -84,20 +82,16 @@ QString BeatFactory::getPreferredSubVersion(
     return (fragments.size() > 0) ? fragments.join(kSubVersionFragmentSeparator) : "";
 }
 
-
-BeatsPointer BeatFactory::makePreferredBeats(
-    const Track& track, QVector<double> beats,
-    const QHash<QString, QString> extraVersionInfo,
-    const bool bEnableFixedTempoCorrection, const bool bEnableOffsetCorrection,
-    const int iSampleRate, const int iTotalSamples,
-    const int iMinBpm, const int iMaxBpm) {
+mixxx::BeatsPointer BeatFactory::makePreferredBeats(
```

Comment:
AFAIK, this method used to build the adequate Beats variety (BeatMap or BeatGrid) according to the available information.
As I'm getting rid of BeatMap and BeatGrid and moving to a common Beats class, my plan is to get rid of the full BeatFactory at some point I will move any needed logic from that class to Beats construction logic. I have not yet removed it because will involve a lot of changes and I want to stabilize first the current code changes.

Reply:
Ah OK, so it sound it will be removed anyway, so no need for a change.